RS-557: SaaS Guide (#150)
reductstore/website: docs/guides/cloud/saas.mdx

The driver has a local cache on a node that stores frequently accessed data, reducing the latency of read and write operations.

The cluster is managed by Google Kubernetes Engine (GKE), which provides automatic scaling, monitoring, and logging for the ReductStore instances.

As a user you don't have access to the cluster and internal components, however you can manage the ReductStore instances using [the SaaS portal](https://cloud.reduct.store).
The portal allows you to create, start, stop, and delete ReductStore instances, as well as monitor the resource usage and access the logs.

## Plan Subscription

Before deploying ReductStore as a SaaS instance, you must register for a ReductStore account and subscribe to a plan.

1. Sign up for a ReductStore account at **[cloud.reduct.store](https://cloud.reduct.store/signup)** with the following information:

   <img
     src={require("@site/docs/guides/cloud/img/saas_signup.png").default}
     alt="Sign up for ReductStore"
     width="500"
     style={{ border: "1px solid #eaecef" }}
   />

> 2. Select a plan that suits your needs and subscribe to it. Click on the **Pricing** tab for a corresponding plan to see more details and subscribe:

Reviewer: ```suggestion
2. Select the plan that best suits your needs (each plan has a **Pricing** tab where you can view details) and complete your subscription:
```
